packages/palette/src/elements/Sup/Sup.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components"
+import { Text, TextProps } from "../Text"
+
+export type SupProps = TextProps
+
+export const Sup = styled(Text)`
+  display: inline;
+  vertical-align: super;
+  font-size: 75% !important;
```

Comment:
Conveniently the new superscript styles are specified as 75% of the current font size. I'm just replacing the old Sup tag with this. It'll be a bit different but not by much.